requirements_dev.txt

```diff
@@ -3,3 +3,4 @@ bumpversion>=0.5.3
 flake8
 pytest>=5.4.1
 setuptools>=41.0.0
+pytest-benchmark
```

Comment:
It would be nice to have histograms using pytes-benchmark, this requires to install `pytest-benchmark[histogram]` instead.